guide/14-deep-learning/how-sequencetosequence-works.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<p>method of the <code><s>TextClassifier</s></code>  <code>SequenceToSequence</code> class</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/869/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>